ami/jobs/management/commands/process_single_image.py

```
@@ -0,0 +1,166 @@
+        job_name = options["name"]
+        wait = options["wait"]
+        poll_interval = options["poll_interval"]
+
+        # Validate image exists
+        try:
+            image = SourceImage.objects.select_related("deployment__project").get(pk=image_id)
+            self.stdout.write(self.style.SUCCESS(f"✓ Found image: {image.path}"))
+            self.stdout.write(f"  Project: {image.deployment.project.name}")
+            self.stdout.write(f"  Deployment: {image.deployment.name}")
+        except SourceImage.DoesNotExist:
+            raise CommandError(f"SourceImage with id {image_id} does not exist")
```

Ruff (0.14.3)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)